config.sh

```diff
@@ -149,8 +149,27 @@ function build_expat {
 }
 
 
+function build_lerc {
+    if [-e lerc-stam ]; then return; fi
+    local cmake=cmake
+    local out_dir=$(fetch_unpack https://github.com/Esri/lerc/archive/refs/tags/v{LERC_VERSION}.tar.gz)
+    (cd $out_dir \ 
+        && mkdir cmake_build && cd cmake_build \
+        && $cmake .. \
+        -DCMAKE_INSTALL_PREFIX=$BUILD_PREFIX \ 
+        -DCMAKE_OSX_DEPLOYMENT_TARGET=$MACOSX_DEPLOYMENT_TARGET \
+        -DBUILD_SHARED_LIBS=ON \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DENABLE_IPO=ON \
+        && $cmake --build . -j4 \
+        && $cmake --install .)
+    touch lerc-stamp
+}
+
+
 function build_tiff {
     if [ -e tiff-stamp ]; then return; fi
+    build_lerc
```

Comment:
we may have to add line like https://github.com/lambgeo/docker-lambda/blob/master/dockerfiles/Dockerfile.gdal3.5#L144-L145

Reply:
@vincentsarago yes, something like that. I actually looked into this over a year ago and found that Lerc wasn't packaged properly and punted.

Reply:
@sgillies I added build_lerc (lines 152-167), looks like LERC fixed their issues with their packaging since then so I think it should be fine

Reply:
okay added those two additional lines @vincentsarago, let me know how that looks. Also should I just download the tar gz to figure out what configuration options libtiff has?